test/api/entry-queryables.spec.ts

```diff
@@ -243,9 +243,17 @@ describe('Query Operators API test cases', () => {
         expect(query.entries[0].created_at).toBeDefined();
       }
     });
+
+    it('should check for include reference', async () => {
+      const query = makeEntries('contenttype_uid').includeReference('abc').query()
+      const result = await query.find<TEntry>()
+      if (query) {
+      console.log("🚀 ~ it ~ query._p:", query._queryParams)
```

Comment:
@harshithad0703 Remove console.log and add proper assertions

Reply:
added